### DISCUSSION TITLE
which version of skywalking is compatible with Istio 1.4.10？

### DISCUSSION THREAD

**Discussion by kuaiwenwu** [apache/skywalking#9674] [Q&A]
Excuse me, which version of skywalking is compatible with Istio 1.4.10, or does the latest version of skywalking support Istio 1.4.10? Is there any missing function?

**Comment by wu-sheng** [comment]
We only tested from Istio 1.8.2.

https://github.com/apache/skywalking/blob/4e73a30d6f42c3293672fb2e5506309650b7393c/.github/workflows/skywalking.yaml#L494

ALS is not stable before that. And Istio 1.4 is super super old release. There is no reason to use that in prod env.
